src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/IHeartbeatDefaultPayloadProvider.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing
+{
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    internal interface IHeartbeatDefaultPayloadProvider
```

Comment:
Please comment explaining what this interface is.

Reply:
Done